src/Pulumi.yaml

```diff
@@ -0,0 +1,5 @@
+name: damavand-test
+runtime:
+  name: python
+  options:
+    virtualenv: ../.venv
```

Comment:
This appears to be different from the virtualenv created by `devenv`. Can you elaborate on why is that necessary to create a separate venv?

Reply:
I commit this by mistake. There is no need for this file. I used it to do manual test as we haven't implemented any automated tests.